ci/pipelines/toolbox-staging-deploy.yml

```diff
@@ -78,17 +93,8 @@ jobs:
             path: /bin/sh
             args:
             - -ec
-            - | 
+            - |
               DIGEST=$(cat toolbox-ecr-registry-staging/digest)
-              assumed_role=($(aws sts assume-role \
-              --role-arn $AWS_ASSUME_ROLE_ARN \
-              --role-session-name ecr-test-session \
-              --query 'Credentials.[AccessKeyId,SecretAccessKey,SessionToken]' \
-              ) )
-
-              export AWS_ACCESS_KEY_ID=${assumed_role[0]}
-              export AWS_SECRET_ACCESS_KEY=${assumed_role[1]}
-              export AWS_SESSION_TOKEN=${assumed_role[2]}
 
               aws configure set aws_access_key_id $AWS_ACCESS_KEY_ID --profile ecr-test-session
```

Comment:
these `aws configure …` lines aren't needed now

Reply:
I did delete these originally, but the build failed. I suspect the `--profile ecr-test-session` bit is key here.

Reply:
which build failed because of this? Can't see it

Reply:
ah so you need to remove https://github.com/alphagov/pay-ci/blob/75683f957e396eba510103704e0208e081d80bf9/ci/pipelines/toolbox-staging-deploy.yml#L103 as well